vision/emergent_object/emergent_object.py

```diff
@@ -0,0 +1,26 @@
+import torch
+import pandas
+
+
+# Function to create model
+def emergent_object_model(path_to_weights):
```

Comment:
Should have the path to your file as a default for the `path_to_weights` parameter